kyc/migrations.py

```diff
@@ -0,0 +1,9 @@
+migrations_list = [
+    ("0", "CREATE TABLE IF NOT EXISTS users (user_id text NOT NULL UNIQUE,email text NOT NULL,verification_code text NOT NULL, verified integer,public_key text NOT NULL,signed_email_identifier text NULL);"),
+    ("0", "CREATE TABLE IF NOT EXISTS migrations (migration_id text NOT NULL);"),
```

Comment:
2x id 0

Reply:
jup, its an example, this doesnt need to happen and will be properly skipped